hnn_core/cell.py

```diff
@@ -162,6 +162,39 @@ def move_to_pos(self):
         """Move cell to position."""
         self.translate_to(self.pos[0] * 100, self.pos[2], self.pos[1] * 100)
 
+    def _connect_feed_at_loc(self, feed_loc, receptor, gid_src, nc_dict,
+                             nc_list):
```

Comment:
What about a test for this new function, in `test_cell.py`? Can a connection be made from two `_Cell` objects b/w the somas?